signup/tool/src/java/org/sakaiproject/signup/tool/jsf/organizer/UserDefineTimeslotBean.java

```diff
@@ -386,21 +389,28 @@ private boolean isEmptyTimeslotWrpList(){
 	public void validateTimeslots(ActionEvent e) {
 		if (this.timeSlotWrpList == null)
 			return;
+		Map<String, String> params = FacesContext.getCurrentInstance().getExternalContext().getRequestParameterMap();
 		int position = 1;
+		int i = 1;
 		for (TimeslotWrapper tsWrp : this.timeSlotWrpList) {
 			if(tsWrp.getDeleted())
 				continue;//skip
 			
-			Map<String, String> params = FacesContext.getCurrentInstance().getExternalContext().getRequestParameterMap();
-
 			String isoStartTime = params.get((position - 1) + HIDDEN_ISO_STARTTIME);
+			String isoEndTime = params.get((position - 1) + HIDDEN_ISO_ENDTIME);
+			
+			if (isoStartTime == null && isoEndTime == null) {
```

Comment:
This method is about validating parameters, is just skipping parameters really accurate? Maybe the location where the parameters are added to the request is actually wrong and this method is correctly identifying that?

Reply:
You have to find the next one isoStartTime and isoEndTime params.
These params don't depend on the position of the Timeslot.
I attach an image: https://snag.gy/tW1V4e.jpg